### ISSUE TITLE
Improve the README and wiki

### ISSUE THREAD

**Issue by almccon** [openterrain/openterrain#1]
I just copied the text from the tumblr, and I think that text was something I copied and lightly modified from the Stamen blog post announcing the Knight grant.

Needs to be rewritten to focus on what the project is about from the point of view of potential contributors.

Then the about text should be copied _back_ to http://openterrain.tumblr.com/projectsummary.

**Comment by almccon** [comment]
In the future we may add new repos in the organization, for things like the processing pipeline. But this repo will remain as the collection of data sources, and the primary landing page for new people curious about the project.

**Comment by ungarj** [comment]
Should we add regional headings (e.g. Europe / Austria) as well to have [local datasets](http://service.salzburg.gv.at/ogdClient/showDetail/d585e816-1a36-4c76-b2dc-6db487d22ca3) better organized?

**Comment by almccon** [comment]
Yes, great idea @ungarj. Currently the [wiki page](https://github.com/openterrain/openterrain/wiki/Terrain-Data) needs a lot of work. None of us have started organizing it yet, so go ahead and get started!

**Comment by ungarj** [comment]
Just added some links & headings. I remembered that UK and the Scandinavian countries released 10m DEMs as well but I was not able to find them yet. Will keep my eyes open though for this and additional sources!

**Comment by ungarj** [comment]
I found some more data sources on [US coastal data](http://coast.noaa.gov/dataviewer/index.html?redirect=301ocm&action=advsearch&qType=in&qFld=projectid&qVal=1005&hc_location=ufi#app=b755&bda3-selectedIndex=0) and from [Spain](http://www.ign.es/ign/layoutIn/modeloDigitalTerreno.do?hc_location=ufi). Both LIDAR derived and high resolution but I wasn't able to find clear information on the license. Shoud we add these sources anyway?

**Comment by sethfitz** [comment]
Let's add them to the list w/ a note to try to clarify what the license is.

**Comment by geowurster** [comment]
I was going to add these to the wiki but someone else was editing at the same time.  I dealt with the LiDAR side of this a lot at my previous job so here's what I know (as of about a year ago):

* I believe the [National Map](http://viewer.nationalmap.gov/viewer/) is working on switching to higher resolution data where available so if you just want DEM's that's probably a good place to start.
* [OpenTopography](http://opentopography.org) primarily hosts small scientific collects but does on demand DEM processing and offers direct FTP access for bulk downloads if you ask.
* NOAA and FEMA have been posting their joint collects on [NOAA Digital Coast](http://coast.noaa.gov/dataregistry/search/collection)
* There's a [list of datasets](http://www.laszip.org/#download-laz-data) on laszip.org
* [USGS EROS](http://eros.usgs.gov/elevation-products) maintains a list of large elevation datasets produced by government entities.
* USGS CLICK's holdings have been migrated to [EarthExplorer](http://earthexplorer.usgs.gov)
* NOAA also maintains a [listing](http://www.coast.noaa.gov/inventory/) of a bunch of elevation sets with pointers to who has the data.
* There's a [Wikipedia page](http://en.wikipedia.org/wiki/National_Lidar_Dataset_(United_States) listing state LiDAR portals
* Some european datasets are being opened up according to [Wikipedia](http://en.wikipedia.org/wiki/National_lidar_dataset)
* I believe there's nationwide data for Germany
* [USGS Data Gateway](https://gdg.sc.egov.usda.gov) has some LiDAR but I suspect it is mirrored elsewhere.

A lot of the datasets in the USGS Interagency Elevation Map haven't been migrated to Earth Explorer but the GIS office holding the data are willing to share if you send a hard drive and pay for shipping.  Some offices charge anywhere between $50 - $500 and others charge a completely ridiculous amount.  In some cases they just have the derivative surface models or the ground returns from the point cloud but in other cases they know exactly what they have and just don't know what to do with it.
